opus/import/do_import.py

```diff
@@ -1204,17 +1205,17 @@ def import_observation_table(instrument_obj,
             if field_type.startswith('flag'):
                 if column_val in [0, 'n', 'N', 'no', 'No', 'NO', 'off', 'OFF']:
                     if field_type == 'flag_onoff':
-                        column_val = 'Off'
```

Comment:
What changed to require this? Isn't mult_label set further down below based on colum_val.title()?